Corgibytes.Freshli.Cli/Functionality/ReadCycloneDxFile.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using PackageUrl;
+
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+public class ReadCycloneDxFile
+{
+    private readonly IReadFile _readFile;
```

Comment:
I think that `_readFile` makes this sound like a method or lambda, because "read" can be interpreted as a verb. I think that this would be a better name:

```suggestion
    private readonly IReadFile _fileReader;
```